webpack.config.js

```diff
@@ -62,32 +63,38 @@ module.exports = {
     console: true,
     fs: 'empty',
     net: 'empty',
-    tls: 'empty'
+    tls: 'empty',
   },
   output: {
-      path:__dirname + "/src/",
-      filename: 'client.min.js',
-   },
+    path: path.join(__dirname, '/src/'),
+    filename: 'client.min.js',
+  },
   plugins: debug ? [
-    new ExtractTextPlugin(`style.css`),
+    new ExtractTextPlugin('style.css'),
     new webpack.ProvidePlugin({
       $: 'jquery',
       jQuery: 'jquery',
       'window.$': 'jquery',
       'window.jQuery': 'jquery',
-      "Hammer": "hammerjs/hammer"
+    }),
+    new Dotenv({
+      path: '.env',
+      safe: true,
     }),
   ] : [
     new webpack.optimize.DedupePlugin(),
     new webpack.optimize.OccurenceOrderPlugin(),
     new webpack.optimize.UglifyJsPlugin({ mangle: false, sourcemap: false }),
-    new ExtractTextPlugin({ filename: `/src/style.css`,  allChunks: true }),
+    new ExtractTextPlugin({ filename: '/src/style.css', allChunks: true }),
     new webpack.ProvidePlugin({
       $: 'jquery',
       jQuery: 'jquery',
       'window.$': 'jquery',
       'window.jQuery': 'jquery',
-      "Hammer": "hammerjs/hammer"
+    }),
+    new Dotenv({
+      path: './.env',
+      safe: true,
```

Comment:
You repeated the newDotenv... and new webpack.ProvidePlugin... twice, I believe this can be refactored.